sections/exchange/TradeCard/CurrencyCard/CurrencyCard.tsx

```diff
@@ -102,6 +110,7 @@ const CurrencyCard: FC<CurrencyCardProps> = ({
 												sign: selectedPriceCurrency.sign,
 										  })
 										: null}
+									{slippagePercent != null && ` (${formatPercent(slippagePercent)})`}
```

Comment:
I think slippagePercent can also be undefined? The percentage is NaN on UI when no wallet is connected.